photon-core/src/main/java/org/photonvision/vision/pipeline/result/CVPipelineResult.java

```diff
@@ -46,8 +47,8 @@ public CVPipelineResult(
     }
 
     public CVPipelineResult(
-            double processingMillis, double fps, List<TrackedTarget> targets, Frame outputFrame) {
-        this(processingMillis, fps, targets, outputFrame, null);
+            double processingNanos, double fps, List<TrackedTarget> targets, Frame outputFrame) {
```

Comment:
If we're using nanoseconds, would a `long` not make more sense?

Reply:
the MathUtils method I use to calculate this returns a double iirc